operators/pkg/controller/elasticsearch/license/apply_test.go

```diff
@@ -241,13 +117,13 @@ func Test_applyLinkedLicense(t *testing.T) {
 			name:    "happy path",
 			wantErr: false,
 			initialObjs: []runtime.Object{
-				&v1alpha1.ClusterLicense{
-					ObjectMeta: k8s.ToObjectMeta(clusterName),
-					Spec: v1alpha1.ClusterLicenseSpec{
-						LicenseMeta: v1alpha1.LicenseMeta{
-							UID: "some-uid",
-						},
-						Type: v1alpha1.LicenseTypePlatinum,
+				&corev1.Secret{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      "test-es-license",
```

Comment:
4 places in this file `test-es-license` is hard-coded, what about `name.LicenseSecret("test")` ?

Reply:
I guess I sometimes think this might help with separating test from production code so that the prod code we are testing is not driving the tests as well. But its a bit of a thin argument.